### ISSUE TITLE
`SegmentationInterpretation.plot_top_losses` ignores `vmin`, keeps it set to 1

### ISSUE THREAD

**Issue by jpunkt** [fastai/fastai#3404]
When using a simple unet for image segmentation with a binary mask (which has only two values, 0 and 1), I noticed that `plot_top_losses` shows the mask and prediction as a solid blue rectangle, although the mask obviously contains something.

I am using fastai v2 in the latest release on my own hardware. OS: Ubuntu 20.04 with CUDA 11.3 and pytorch 1.8.1

*How to reproduce*:
- Create a new unet learner using the high-level API (as described [here](https://docs.fast.ai/tutorial.vision.html#Segmentation---Using-the-high-level-API))
- Use custom generated `ImageMask`s, with only two values, 0 and 1.
- After training the model, create a `SegmentationInterpretation` using the `from_learner` function.
- View the output of `plot_top_losses`. The masks and predictions look blank when they are not.

*Expected behaviour*:
- When using `SegmentationDataLoaders.show_batch` with the same mask data, I can pass an argument `vmin=0`. This causes the values labeled with 1 to be displayed in color.
- Not passing `vmin=0` also shows the mask as blank (see also [this forum post](https://forums.fast.ai/t/segmentationdataloaders-show-batch-not-showing-mask-in-other-color/66071/2))
- I would expect the same behaviour for `SegmentationInterpretation.plot_top_losses`. However, I can pass the argument `vmin=0`, but it is ignored.

*Known Workaround*:
- I could mitigate the problem by using the value 2 in my `ImageMask` instead of 1. While this works, I find it irritating. Being able to use the same parameters as for `show_batch` would be more consistent.  I also seem (?) to have to include a third bogus value in my `codes` list for the dataloader to avoid `AssertionErrors` when training the learner, however I was not able to verify this.

**Comment by jpunkt** [comment]
*Update:*

- Using a maximum value of 2 in the mask forces me to provide a list of `codes` with 3 entries, even if every mask only contains two values. Not doing so triggers `RuntimeError`s on training. The reason given for the error is mostly different, but providing three values in `codes` consistently fixes the issue.
- I tried using 1 and 2 as values in the mask instead of 0 and 1. This does not change the `RuntimeError`s being triggered when training as described above.

**Comment by tcapelle** [comment]
I will take a look at this tomorrow.
